lib/connextclient/ConnextClient.ts

```diff
@@ -692,6 +694,44 @@ class ConnextClient extends SwapClient {
     });
   }
 
+  public withdraw = async ({
```

Comment:
Can be done in a follow-up PR/issue - it would be nice to get a test case (or two) for this new code as it contains quite a few conditionals.